packages/mdx-embed/src/components/youtube/youtube.test.tsx

```diff
@@ -20,6 +20,17 @@ describe('<YouTube />', () => {
     expect(screen.getByTestId('youtube')).toBeDefined();
   });
 
+  test('it renders the component when provided a listId', () => {
```

Comment:
if you do make the change to the prop you might also need to change the description for the test